docs/developer/guidelines/dashboards.md

```diff
@@ -54,128 +49,115 @@ Ensure logos render correctly on the Dashboard List page and within the preset d
 1. Research the metrics supported by the integration and consider grouping them in relevant categories.
 Important metrics that are key to the performance and overview of the integration should be at the top.
 
-1. Ensure good design by keeping labels and widgets aligned with consistent spacing and reasonable size.
+### Rules of Thumb
 
-!!! tip
-    Use [`TV Mode`](https://docs.datadoghq.com/dashboards/screenboards/#tv-mode) to ensure that the whole dashboard fits your screen.
-    As more content is added, the recommended view of preset dashboards is vertical.
+1. Always include an “about” group for the integration containing a brief description and helpful links. Also include an “overview” group of a few of the most important metrics at the top of a dashboard. Use a group banner image for the “about” section if one is available. We store banner images [here](https://github.com/DataDog/web-ui/tree/preprod/public/static/images/integration_dashboard) and link to them with relative links in the dashboard. The “about” section should contain content, not data; the “overview” section should contain data. Avoid making the “about” section full-width.
 
-### Logos
+![Elasticsearch dashboard](https://raw.githubusercontent.com/DataDog/integrations-core/master/docs/developer/assets/images/example_dashboard.png)
```

Comment:
These images aren't working in my preview; I think direct links to `raw.githubusercontent.com` usually contain `blob/`right before the `master/` part of the URL, but it probably depends on how this gets published (which I don't know about).

Reply:
They won't render at these URLs until they're merged. The URL should work since the current guide uses an image from the same directory with this URL structure: `https://raw.githubusercontent.com/DataDog/integrations-core/master/docs/developer/assets/images/ignite_dash.jpeg`